### PR TITLE
Update minor

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md [hunk 1]
```diff
@@ -14,10 +14,9 @@ Please refer to [Index Documentation](index.md) on details about index.
 
 You can use the `If EXISTS` keywords when dropping tags. These keywords automatically detect if the corresponding tag exists. If it exists, it will be deleted. Otherwise, no tag is deleted.
 
-A vertex can have either only one tag (types) or multiple tags (types).
+A vertex can have one to more tags (types).
```
Comment (oldLady344) [inline docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md:17]: one or more

File: docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md [hunk 2]
```diff
@@ -14,10 +14,9 @@ Please refer to [Index Documentation](index.md) on details about index.
...
 
-In the former case, such a vertex can NOT be accessible after the statement, and edges connected with such vertex may result in DANGLING.
-
-In the latter case, the dropped a vertex is still accessible. But all the properties defined by this dropped tag are not accessible.
+If vertices have only one tag, after dropping the tag, vertices labelled with this tag can NOT be accessible. But the edges connected with the vertices are available.
```
Comment (oldLady344) [inline docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md:19]: labeled
https://www.grammarly.com/blog/labeled-labelled/

"the edges connected with the vertices" ==> "the edges connecting the vertices"

File: docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md [hunk 3]
```diff
@@ -14,10 +14,9 @@ Please refer to [Index Documentation](index.md) on details about index.
...
+If vertices have multiple tags, after dropping one of the tags, the vertices are still accessible. But all the properties defined by this dropped tag are not accessible.
```
Comment (oldLady344) [inline docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md:20]: If a vertex has multiple tags, after one tag is dropped, the vertex is still accessible. 

singularity may make this sentence more understandable.

File: docs/manual-EN/3.build-develop-and-administration/4.account-management-statements/built-in-roles.md [hunk 4]
```diff
@@ -56,4 +56,4 @@ Divided by operations.
 | Special operation | Y | Y | Y | Y | Y |
 | Show Jobs | Y | Y | Y | Y | Y |
 
-Note: Pay attention to the special operation here. The returned results vary based on the account role. For example, each role have the access to the `SHOW SPACE` statement, but the returned results vary based on the account authentication.
+**Note:** Pay attention to the special operation here. The returned results vary based on the account role. For example, each role have the access to the `SHOW SPACE` statement, but the returned results vary based on the account authentication. Only the `GOD` user have the authority to run the `SHOW USERS` and the `SHOW SNAPSHOTS` statement.
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/4.account-management-statements/built-in-roles.md:59]: "each role have the access" ==> "each role has access "

"Only the `GOD` user have the authority to run the `SHOW USERS` and the `SHOW SNAPSHOTS` statement." ==> "Only the `GOD` user has the authority to run the `SHOW USERS` and the `SHOW SNAPSHOTS` statements.

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md [hunk 5]
```diff
@@ -2,12 +2,12 @@
 
 ## Overview
 
-Spark Writer is Nebula Graph's Spark-based distributed data import tool that converts data from multiple data repositories into vertices and edges of graphs and batch imports data into the graph database. Currently supported data repositories are:
+Spark Writer is a Spark-based distributed data importer for [Nebula Graph](https://github.com/vesoft-inc/nebula). It converts data from multiple data sources into vertices and edges for graphs and batch imports data into the graph database. Currently supported data sources are:
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md:5]: Currently, the supported data sources are:

Or change it to "It supports source data of these formats:"

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md [hunk 6]
```diff
@@ -2,12 +2,12 @@
...
 
 * HDFS, including Parquet, JSON, ORC and CSV
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md:7]: HDFS (including Parquet), JSON, ORC, and CSV?

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md [hunk 7]
```diff
@@ -54,7 +54,7 @@ Note: Please create a space and define the schema in Nebula Graph first, then us
 
 #### Vertices
 
-A vertex data file consists of multiple rows, with each line in the file representing a point and its properties. In general, the first column is the ID of the vertex. This ID column is specified in the mapping file. Other columns are the properties of the vertex. Consider the following example in JSON format.
+A vertex data file consists of multiple rows. Each line in the file represents a vertex and its properties. In general, the first column is the ID of the vertex. This ID column is specified in the mapping file. Other columns are the properties of the vertex. Consider the following example in JSON format.
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md:57]: represents one vertex and its properties. 

"Other columns" ==> "The other columns"

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md [hunk 8]
```diff
@@ -66,7 +66,7 @@ A vertex data file consists of multiple rows, with each line in the file represe
 
 #### Edges
 
-An edge data file consists of multiple rows, with each line in the file representing a point and its properties. In general, the first column is the ID of the source vertex, the second column is the ID of the destination vertex. These ID columns are specified in the mapping file. Other columns are the properties of the edge. Consider the following example in JSON format.
+An edge data file consists of multiple rows. Each line in the file represents a point and its properties. In general, the first column is the ID of the source vertex, the second column is the ID of the destination vertex. These ID columns are specified in the mapping file. Other columns are the properties of the edge. Consider the following example in JSON format.
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md:69]: "represents a point and its properties."==>"represents one point and its properties."

"Other columns" ==> "The other columns"

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md [hunk 9]
```diff
@@ -153,11 +153,11 @@ Player format as follows:
 
 ### Write Configuration Files
 
-The configuration files consist of Spark related information, Nebula Graph related information, and tags mapping and edges mapping blocks. Spark information is configured with the associated parameters running Spark. Nebula Graph information is configured with information such as user name and password to connect Nebula Graph. Tags mapping and edges mapping correspond to the input source mapping of multiple tag/edges respectively, describing the basic information like each tag/edge's data source. It's possible that different tag/edge come from different data sources.
+The configuration files consist of the Spark field, the Nebula Graph field, the tags mapping field, and the edges mapping field. The Spark related parameters are configure in the Spark field. The username and password information for Nebula are configured in the Nebula field. Basic data source information for each tag/edge is described in the tag/edge mapping field. The tag/edge mapping field corresponds to multiple tag/edge inputting sources. Different tag/edge can come from different data sources.
```
Comment (oldLady344) [inline docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md:156]: "The Spark related parameters are configure in the Spark field." ==> "The Spark related parameters are configured in the Spark field."

"for Nebula" ==> "for Nebula Graph"?

"in the Nebula field" ==> "in the Nebula Graph field"

"tag/edge" ==> "tag or edge"

File: docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md (hunk 6 above)
Comment (Amber1990Zhang) [reply docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md:7]: 是HDFS包括这几种格式的文件

**Comment by Amber1990Zhang** [comment]
https://github.com/vesoft-inc/nebula-docs/pull/127
fixed here

**Pull request merged**